rxandroidble/src/main/java/com/polidea/rxandroidble2/RxBleConnection.java

```diff
@@ -238,7 +240,7 @@ public String toString() {
      * NOTE: due to stateful nature of characteristics if one will setupIndication() before setupNotification()
```

Comment:
NOTE: due to stateful nature of characteristics, if you call setupIndication() before setupNotification()